### DISCUSSION TITLE
What are the best statistical methods we can add in the next application?

### DISCUSSION THREAD

**Discussion by amandalemette** [amandalemette/PyStat#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ldsnetto** [comment]
We can add Pearson correlation, spearman correlation, boxplot, and some features that from seaborn library: like the paiplot for example.

**Reply by amandalemette** [reply]
ok, I liked the suggestions, let's do it!
